parsers/ShortcodeParser.php

```diff
@@ -612,4 +620,10 @@ function ($matches) use ($tags, $parser) {
 
 		return $content;
 	}
+
+	public static function getShortcodes($instance = 'default')
```

Comment:
Oops.
I added that method for debugging what the hell was going on with HTMLEditorFieldTest failing. Forgot to remove it. Indeed we don't need it at all.